pkg/paymentMethod.go

```diff
@@ -1,13 +1,69 @@
 package moov
 
-type AutoGenerated struct {
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+type PaymentMethod struct {
 	PaymentMethodID   string `json:"paymentMethodID,omitempty"`
 	PaymentMethodType string `json:"paymentMethodType,omitempty"`
 	Wallet            Wallet `json:"wallet,omitempty"`
 }
 
 // ListPaymentMethods lists all payment methods that are associated with a Moov account
 // https://docs.moov.io/api/index.html#tag/Payment-methods/operation/getPaymentMethods
+func (c Client) ListPaymentMethods(accountID string, sourceID string) ([]PaymentMethod, error) {
+	var resPaymentMethods []PaymentMethod
+	url := fmt.Sprintf("%s/%s", baseURL, fmt.Sprintf(pathPaymentMethods, accountID))
```

Comment:
Why are we building URLs like this? It seems overly complicated.

Reply:
Suggestion? We also have this URL structure but that is easy for an append to the string vs inserting the {accountID}

https://github.com/moovfinancial/moov-go/blob/main/pkg/account.go#L404